src/site/content/en/animations/animations-examples/index.md

```diff
@@ -0,0 +1,346 @@
+The `<div>` with a class of `pulsating-circle` marks the position of the circle,
+however it does not draw a circle itself.
+
+```css
+.pulsating-circle {
+    position: absolute;
+    left: 50%;
+    top: 50%;
+    transform: translateX(-50%) translateY(-50%);
+    width: 30px;
+    height: 30px;
+}
+```
+
+The visible circle and animations are achieved using the `::before` and `::after` pseudo-elements.
+
+The `::before` element creates the opaque ring that extends outside of the white circle,
+using an animation called `pulse-ring`,
+which animates `transform: scale`, and `opacity`.
+
+```css
+.pulsating-circle::before {
+    content: '';
+    position: relative;
+    display: block;
+    width: 300%;
+    height: 300%;
+    box-sizing: border-box;
+    margin-left: -100%;
+    margin-top: -100%;
+    border-radius: 45px;
+    background-color: #01a4e9;
+    animation: pulse-ring 1.25s cubic-bezier(0.215, 0.61, 0.355, 1) infinite;
+}
+
+@keyframes pulse-ring {
+  0% {
+    transform: scale(0.33);
+  }
+  80%, 100% {
+    opacity: 0;
+  }
+}
+```
+
+Another way to see which properties are being animated is to select the **Animations** panel in Firefox DevTools.
+You will then see a visualization of the animations being used,
+and the properties that are being animated.
+
+<figure class="w-figure">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/animation-circle-firefox-devtools.mp4" type="video/webm">
+  </video>
+  <figcaption class="w-figcaption">
+    With the ::before pseudo-element selected we can see which properties are animating.
+  </figcaption>
+</figure>
+
+The white circle itself is created and animated using the `::after` pseudo-element.
+The animation `pulse-dot` uses `transform: scale` to grow and shrink the dot during the animation.
+
+```css
+.pulsating-circle::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: block;
+  width: 100%;
+  height: 100%;
+  background-color: white;
+  border-radius: 15px;
+  box-shadow: 0 0 8px rgba(0, 0, 0, 0.3);
+  animation: pulse-dot 1.25s cubic-bezier(0.455, 0.03, 0.515, 0.955) -0.4s infinite;
+}
+
+@keyframes pulse-dot {
+  0% {
+    transform: scale(0.8);
+  }
+  50% {
+    transform: scale(1);
+  }
+  100% {
+    transform: scale(0.8);
+  }
+}
+```
+
+An animation like this could be used in various places in your application,
+it's important that these small touches don't impact the overall performance of your app.
+
+## Pure CSS 3D Sphere
+
+{% IFrame {
+  src: 'https://codepen.io/iamlark/embed/jYzYJg?height=458&theme-id=light&default-tab=result',
+  title: 'Pure CSS 3d Sphere'
+}
+%}
+
+This animation seems incredibly complicated,
+however it uses techniques that we have already seen in the previous examples.
+The complexity comes from animating a large number of elements.
+
+Open Chrome DevTools and select on of the elements with a class of `plane`.
```

Comment:
It might be nice to mention the DOM Tree search feature for cases where you know the elements you're looking for: https://youtu.be/VYyQv0CSZOE?t=253

In this case searching for `.plane`

https://developers.google.com/web/tools/chrome-devtools/dom#search